modules/ocf_desktop/manifests/firewall_output.pp

```diff
@@ -0,0 +1,24 @@
+# allow desktops to send packets to papercut, pagefault, radiation
```

Comment:
Why to radiation? Do the desktops ever need to communicate with it?

Reply:
(I know this was asked before, but I don't believe it was answered)

Reply:
Never mind, it was answered, but not in the same thread. I guess the access point's configuration interface is useful to have access to, although I've never actually used it, but I'm fine with allowing it to be accessible.